readthedocs/projects/models.py

```diff
@@ -1825,6 +1828,11 @@ class EnvironmentVariable(TimeStampedModel, models.Model):
         on_delete=models.CASCADE,
         help_text=_('Project where this variable will be used'),
     )
+    public = models.BooleanField(
+        _('Public'),
+        default=False,
+        help_text=_('Expose this environment variable in PR builds?')
+    )
```

Comment:
When adding a new field we mark it a `null=True`, this is to avoid downtime when doing the deploy. For example https://github.com/readthedocs/readthedocs.org/blob/f18c584d9e6ba71105a0c42753ed6e9fbe4ebf0a/readthedocs/projects/models.py#L1379-L1384

And when adding it to the form we need to remove the none option

https://github.com/readthedocs/readthedocs.org/blob/f18c584d9e6ba71105a0c42753ed6e9fbe4ebf0a/readthedocs/projects/forms.py#L229-L231

Reply:
Ah, so that’s what’s missing from the migration!